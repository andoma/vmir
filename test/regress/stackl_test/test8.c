//#include <string.h>

int f2();
int f1(int a, int b);

int f1(int a, int b)
{
    printf("%d", a + b);
    puts("\n");
    return a + b;
}
int main()
{
    int ret;
    printf("%d", 0);
    puts("\n");
    f1(5, 8);
    ret = f2();
    printf("%d", 3);
    puts("\n");
    printf("%d", ret);
    puts("\n");
    ret = 2*f2() - 3;
    printf("%d", ret);
    puts("\n");
}
int f2()
{
    printf("%d", 2);
    puts("\n");
    return 4;
}
