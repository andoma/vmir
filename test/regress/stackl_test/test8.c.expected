0
13
2
3
4
2
5

